BackEnd/api.py

```diff
@@ -2,21 +2,16 @@
 import json
 client = OpenAI()
 
-json_obj = ""
-def get_response(message):
+def lambda_handler(event, context):
+    message = event["message"]
+    meta = event["meta"]
     response = client.chat.completions.create(
         model="gpt-3.5-turbo-1106",
         response_format={"type":"json_object"},
         temperature=0.7,
-        messages=[{"role":"system", "content":"You are a helpful assistant that extracts data and returns it in JSON format. Need to format as \"input\": qustion, \"result\": answer "},
+        messages=[{"role":"system", "content":"You are a helpful assistant that extracts data and returns it in JSON format. Need to format as \"response\": answer "},
                   {"role":"user","content":message}],
         max_tokens=100
     )
-    return response.choices[0].message.content
-
-message = input()
-response = get_response(message)
-
-json_obj = json.loads(response)
-
-print(json_obj)
\ No newline at end of file
+    json_obj = json.loads(response.choices[0].message.content)
```

Comment:
It is better not to define unused variables

Reply:
Agree. That would cause some unnecessary storing space.